.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+  - package-ecosystem: bundler
+    directory: "/"
+    registries: "*"
+    schedule:
+      interval: weekly
+      day: monday
+      time: "08:00"
+      timezone: PST8PDT
+    open-pull-requests-limit: 10
+    versioning-strategy: auto
+    commit-message:
+      prefix: No-Jira
+      include: scope
+    groups:
+      minor-patch-versions: # Group together minor / patch updates.
+        applies-to: "version-updates"
+        update-types:
+          - "minor"
+          - "patch"
+      minor-patch-security:
+        applies-to: "security-updates"
+        update-types:
+          - "minor"
+          - "patch"
```

Comment:
We haven't spent time spiking a better solution for dependabot so we should skip adding this file and the dependabot_automerge.yml for now